### PR TITLE
Device Plugins: add info about GA graduation

### PR DESCRIPTION
**Pull request by bart0sh**
This is a documentation update for the [Add CDI devices to device plugin API](https://github.com/kubernetes/enhancements/issues/4009) feature. The feature is planned for GA graduation in 1.31.

The documentation update was explicitly requested in [this comment](https://github.com/kubernetes/enhancements/issues/4009#issuecomment-1943582878)

### CODE REVIEW

**Comment by reylejano** [comment]
If the https://github.com/kubernetes/enhancements/issues/4009 feature is planned for 1.30 then this should target the `dev-1.30` branch

**Comment by bart0sh** [comment]
@reylejano rebased on top of `dev-1.30`, thanks for pointing out.

**Comment by drewhagen** [comment]
/hold
@bart0sh Kubernetes 1.30 is releasing today.
This PR will need to be targeting the `dev-1.31` branch once it is available. Our SIG Release docs team will keep you posted.

**Comment by drewhagen** [comment]
You can reopen this in the next release cycle and target `dev-1.31`. Thanks! :)

**Comment by drewhagen** [comment]
Heads up that `dev-1.31` branch is created now, if you choose to reopen this for the next release.

**Comment by bart0sh** [comment]
@drewhagen 
> Heads up that dev-1.31 branch is created now, if you choose to reopen this for the next release.

@kubernetes/website-milestone-maintainers Please, set milestone v1.31 for this PR, thanks.

**Comment by bart0sh** [comment]
@klueska As the [code PR](https://github.com/kubernetes/kubernetes/pull/123315) has been merged, can you review this one, please?

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md
Comment (tengqm) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:163]:
```suggestion
   v1.28, graduated to beta in v1.29 and to GA in v1.31.
```

Comment (tengqm) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:163]: You will also need to update the file under `content/en/docs/reference/command-line-tools-reference/feature-gates/`.

Comment (bart0sh) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: done, thanks

**Comment by bart0sh** [comment]
@tengqm thank you for the review. All review comments should be addressed, PTAL again.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/device-plugin-cdi-devices.md
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/device-plugin-cdi-devices.md:16]:
```suggestion
    toVersion: "1.30"
```

Comment (bart0sh) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/device-plugin-cdi-devices.md]: Done, thanks.

**Comment by bart0sh** [comment]
@klueska @elezar can you lgtm?

**Review by elezar — APPROVED** [review]
/lgtm

**Pull request merged**
